src/pheval/analyse/generate_summary_outputs.py

```diff
@@ -34,29 +34,11 @@ def _calculate_rank_difference(self) -> pd.DataFrame:
         comparison_df["rank_decrease"] = comparison_df.iloc[:, 3] - comparison_df.iloc[:, 2]
         return comparison_df
 
-    def generate_gene_output(self, prefix: str) -> None:
-        """Generate the output for gene prioritisation ranks."""
-        self._generate_dataframe().to_csv(prefix + "-gene_rank_comparison.tsv", sep="\t")
+    def generate_output(self, prefix: str, suffix: str):
```

Comment:
You are a tiny bit inconsistent in whether a function that does not return anything should end in `-> None` or not